### DISCUSSION TITLE
Has github actions environments been removed?

### DISCUSSION THREAD

**Discussion by JamesArthurHolland** [community/community#24935] [Actions]
<ol>
<li>On GitHub, navigate to the main page of the repository.</li>
<li>Under your repository name, click <strong>Settings</strong> .</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/4/84a64a987aeab2af381330e77d099a2b1985d4af.png" data-download-href="/uploads/short-url/iVtiPDTHMeZmYZhjWkqaJNDpRW7.png?dl=1" title="Repository settings button" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10409492/181087814-48dd4f41-29ea-44fd-b590-b60351ad3a21.png" alt="Repository settings button" data-base62-sha1="iVtiPDTHMeZmYZhjWkqaJNDpRW7" width="690" height="122" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a64a987aeab2af381330e77d099a2b1985d4af_2_690x122.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a64a987aeab2af381330e77d099a2b1985d4af_2_1035x183.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a64a987aeab2af381330e77d099a2b1985d4af_2_1380x244.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a64a987aeab2af381330e77d099a2b1985d4af_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Repository settings button</span><span class="informations">1552×276 21.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<ol start="3">
<li>In the left sidebar, click <strong>Environments</strong> .</li>
<li>Click <strong>New environment</strong> .</li>
</ol>
<p>I can’t find “Environments” in the left side bar</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Based on the following quote from the documentation I assume it’s because it’s a private repository:</p>
<blockquote>
<p>Environments, environment protection rules, and environment secrets are available in public repositories for all products and in private repositories for GitHub Enterprise.</p>
</blockquote>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/actions/reference/environments" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/actions/reference/environments" target="_blank" rel="noopener">Environments - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/actions/reference/environments" target="_blank" rel="noopener">//docs.github.com/en/actions/reference/environments</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
